ru/mail/chrome/messenger/messengercompose/messengercompose.dtd

```diff
@@ -10,15 +10,15 @@
 <!ENTITY newMessage.key "M">
 <!ENTITY newMessageCmd2.key "N">
 <!ENTITY newMessage.accesskey "M">
-<!ENTITY newCard.label "Address Book Card...">
+<!ENTITY newCard.label "Карточку адресной книги...">
 <!ENTITY newCard.accesskey "C">
 <!ENTITY emptyAttachmentBucket.label "Добавить файлы...">
 <!ENTITY attachMenu.label "Вложить">
 <!ENTITY attachMenu.accesskey "t">
-<!ENTITY attachFileCmd.label "Файл(ы)…">
+<!ENTITY attachFileCmd.label "Файл(ы)...">
 <!ENTITY attachFileCmd.accesskey "F">
-<!ENTITY attachPageCmd.label "Веб-страницу…">
-<!ENTITY attachPageCmd.accesskey "-">
+<!ENTITY attachPageCmd.label "Веб-страницу...">
+<!ENTITY attachPageCmd.accesskey "W">
```

Comment:
@yshchebet  Please revert this line.  If we're missing a hotkey, then let us know, but please do not change hotkeys.  It could cause problems elsewhere in the application.